components/RecipeForm/IngredientInput.tsx

```diff
@@ -6,10 +6,13 @@ import {
 } from '@/components/RecipeForm/recipeStyles';
 import { IngredientInputProps } from '@/types/IngridientInput.types';
 import { HandleChangeParams } from '@/types/RecipeForm.types';
+import DeleteIcon from '@/public/icons/buttons/ic_fluent_delete_24_regular.svg';
+import { StyledDeleteIconWrapper } from '@/components/RecipeForm/recipeStyles';
```

Comment:
It would be better to call this "StyledDeleteIngredientButton" because - its a button :)

Reply:
You're right, thanks! :)